src/lib/comms/sol-lwm2m.c

```diff
@@ -3008,9 +3008,103 @@ clear_bootstrap_ctx(struct sol_lwm2m_client *client)
     }
 }
 
+static int
+is_authorized(struct sol_lwm2m_client *client,
```

Comment:
I guess we should document here that 1 means authorized, 0 means not authorized and < 0 mean error.